american-journal-of-science.csl

```diff
@@ -0,0 +1,201 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-US">
+  <info>
+    <title>American Journal of Science</title>
+    <title-short>AJS</title-short>
+    <id>http://www.zotero.org/styles/american-journal-of-science</id>
+    <link href="http://www.zotero.org/styles/american-journal-of-science" rel="self"/>
+    <link href="http://www.nwrc.usgs.gov/techrpt/sta28.pdf" rel="documentation"/>
```

Comment:
sorry. I started all right and then something effed up and I went back to the template and I forgot to change these things. very sorry.